torch/distributions/utils.py

```diff
@@ -94,7 +96,7 @@ class lazy_property(object):
     """
     def __init__(self, wrapped):
         self.wrapped = wrapped
-        update_wrapper(self, wrapped)
+        update_wrapper(self, wrapped)  # type: ignore[arg-type]
```

Comment:
is there a link to the reason?

Reply:
Fixing this error requires annotating a built-in python package(functools), therefore I just ignore this error.
More details: update_wrapper(wrapper: [Callable, ...], wrapped) is a functools function. Here the Argument 1 is "self", which is "class lazy_property", therefore the type mismatches.

Reply:
To fix this problem we need to annotate the update_wrapper(wrapper, wrapped), which is in a third-party python package(functools). The function's first parameter is "[Callable, ...]" whereas the Argument 1 here("self") is type "class lazy_property(object)".